src/box/memtx_tx.c

```diff
@@ -1048,8 +1063,8 @@ memtx_tx_tuple_clarify_slow(struct txn *txn, struct space *space,
 			break;
 		}
 	}
-	if (!own_change)
-		memtx_tx_track_read(txn, space, tuple);
+	if (!own_change && result != NULL)
```

Comment:
Is it possible to cover case when result != NULL and own_change == true? I just don't like that removing `!own_change` doesn't cause test crashes. But I guess this condition might be just an optimization?